airflow/models.py

```diff
@@ -2031,20 +2008,32 @@ def date_range(self, start_date, num=None, end_date=datetime.now()):
             start_date=start_date, end_date=end_date,
             num=num, delta=self._schedule_interval)
 
-    def following_schedule(self, dttm):
+    def directly_following_schedule(self, dttm):
         if isinstance(self._schedule_interval, six.string_types):
             cron = croniter(self._schedule_interval, dttm)
             return cron.get_next(datetime)
         elif isinstance(self._schedule_interval, timedelta):
             return dttm + self._schedule_interval
 
-    def previous_schedule(self, dttm):
+    def following_schedule(self, dttm, n=1):
```

Comment:
I would add documentation for n here, and maybe rename it as num_executions or so